utils/data.ts

```diff
@@ -81,7 +81,7 @@ export const transactionTabs: TransactionData = {
       lazlo: 'pending',
     },
     input: {
-      sats: '101,000',
+      sats: '100,000',
     },
     output_0: {
       sats: '100000',
```

Comment:
Same comment as above, I believe this should be 99,000 sats (related to https://github.com/saving-satoshi/saving-satoshi/issues/1175)

Reply:
fixed in https://github.com/saving-satoshi/saving-satoshi/commit/788751e65ddc028de04bf8d084fbd12a9c3f5890